book/animations.md

```diff
@@ -1961,136 +1961,17 @@ and compositing instead.
-so they can't cause additional overlap either (though they might cause *less*
-overlap).
-
-But this is both unsatisfying and boring, because in fact there *are* visual
-effects that can cause additional overlap. The most important is *transforms*,
-which are a mechanism to move around the painted output of a DOM element
-anywhere on the screen.[^blur-filter] In addition, transforms are one of the
-most popular visual effects in browsers, because they allow you to move around
-content efficiently on the GPU and the browser thread. That's because
-transforms merely apply a linear transformation matrix to each pixel, which is
-one of the things GPUs are good at doing efficiently.[^overlap-example]
-
-[^blur-filter]: Certain [CSS filters][cssfilter], such as blurs, can also expand
-pixel rects.
-
-[cssfilter]: https://developer.mozilla.org/en-US/docs/Web/CSS/filter
-
-[^overlap-example]: In addition, it's not possible to create the overlapping
-squares example of this section without something like transforms. Real
-browsers have many other methods, such as [position]. In fact, it's a bit
-difficult to cause overlap at all in our current browser, though one way is to
-set the `width` and `height` of elements such that it causes text to overflow
-on top of siblings further down the page.
-
-[position]: https://developer.mozilla.org/en-US/docs/Web/CSS/position
-
-::: {.further}
-
-Overlap reasons for compositing not only create complications in the code, but
-without care from the browser and web developer can lead to a huge amount of
-GPU memory usage, as well as page slowdown to manage all of the additional
-composited layers. One way this could happen is that an additional composited
-layer results from one element overlapping another, and then a third because it
-overlaps the second, and so on. This phenomenon is called *layer explosion*.
-Our browser's algorithm avoids this problem most of the time because it is able
-to merge multiple paint chunks together as long as they have compatible
-ancestor effects, but in practice there are complicated situations where it's
-hard to make content merge efficiently.
-
-In addition to overlap, there are other situations where compositing has
-undesired side-effects leading to performance problems. For example, suppose we
-wanted to *turn off* composited scrolling in certain situations, such as on a
-machine without a lot of memory, but still use compositing for visual effect
-animations. But what if the animation is on content underneath a scroller? In
-practice, it is very difficulty to implement this situation correctly without
-just giving up and compositing the scroller.
-
-:::
-
 
 Transform animations
 ====================
 
-The `transform` CSS property lets you apply linear transform visual
+So far, we've been putting two paint commands in the same layer if
+they have the same effects applied. This works---but if paint commands
```

Comment:
"if paint commands overlap"?

The reason I didn't change the order before (even though you suggested it in previous meetings :) ) was that it felt better to properly motivate overlap and why it matters before introducing a method of overlapping. Seems you think the other way around is better...let's just decide together at our meeting today.

Reply:
Ok sounds good.